blog/drafts/ultimate-commands-guide/ultimate-commands-guide.md

```diff
@@ -0,0 +1,313 @@
+# Ultimate validator maintenance guide. Testnet: Euler-1.
+
+## Glossary
+
+  **Bandwidth** - 1 bandwidth of approximately 55 600 000 cyb. The recovered unit of your account. Depends on your cyb count. Used to complete transactions in the cyberd blockchain. Transaction costs from 8 to 800 bandwidth - recalculate by price sliding window. Full regeneration time 24 hours
+
+  **commission** -  tokens that you've earned with validation. You can take them at any time.
+
+  **illiquid tokens** - non-transferable tokens that you've delegated to the validator. Delegation process duration - 1 block. **Unbonding** process, or taking back share - 3 weeks.
+
+  **link** - reference between CID key and CID value. ransaction costs from 8 to 800 bandwidth - recalculate by price sliding window. Linking time 100 s.
```

Comment:
With `price=1`:
Link message cost is `100*n`, where `n` is quantity of links in message.
Other messages cost is `500`.
Transaction consists of one or more messages `m_1, m_2, ..., m_n`. Transaction cost is `300 + c_1 + c_2 ... + c_n`, where `c_i` - cost of `m_i` message.